src/sentry/integrations/slack/notifications.py

```diff
@@ -21,4 +24,30 @@ def send_notification_as_slack(
         status=ObjectStatus.VISIBLE,
     )
     for integration in integrations:
-        pass
+        attachment = [build_notification_attachment(notification, context)]
+        external_actor = ExternalActor.objects.get(
+            integration=integration,
+            provider=ExternalProviders.SLACK.value,
+            actor=user.id,
+            organization=notification.organization,
+        )
+        payload = {
+            "token": integration.metadata["access_token"],
+            "channel": external_actor.external_id,
+            "link_names": 1,
+            "attachments": json.dumps(attachment),
+        }
+        client = SlackClient()
+        try:
+            client.post("/chat.postMessage", data=payload, timeout=5)
+        except ApiError as e:
+            logger.info(
+                "notification.fail.slack_post",
+                extra={
+                    "error": str(e),
+                    "notification": notification,
+                    "user": user.id,
+                    "channel_id": external_actor.external_id,
+                },
+            )
+        metrics.incr("notifications.sent", instance="slack.notification", skip_internal=False)
```

Comment:
This is a copy/paste from when we send alert rules but I wasn't sure what to change the metrics name to instead.